packages/commonwealth/server/workers/evmChainEvents/emitChainEventNotifs.ts

```diff
@@ -0,0 +1,76 @@
+import { RawEvmEvent } from './types';
+import { ChainAttributes } from '../../models/chain';
+import { ContractAttributes } from '../../models/contract';
+import models from '../../database';
+import { QueryTypes } from 'sequelize';
+import { NotificationDataAndCategory } from 'types';
+import { NotificationCategories } from 'common-common/src/types';
+import { SupportedNetwork } from 'chain-events/src';
+import emitNotifications from '../../util/emitNotifications';
+import { rollbar } from './startEvmPolling';
```

Comment:
This import is causing a type error in CI as rollbar is not exported from ./startEvmPolling

Reply:
Looks like this was fixed in the above commit(commented right as the commit came in)